core/templates/dev/head/pages/learner_dashboard/learner_dashboard.html

```diff
@@ -661,8 +661,15 @@ <h1 class="stat-value"><[subscriptionsList.length]></h1>
                   <span style="margin-left: 7px; font-size: 105%;"><[explorationTitle]></span>
                 </td>
               </tr>
-
-              <tr ng-repeat="message in messageSummaries" style="padding: 13px 8px 7px 8px;">
+              <tr ng-if="loadingFeedbacks">
+                <td style="text-align: center;">
+                  <span>
+                    Loading
```

Comment:
Minor note: could make this slightly longer, maybe "Loading feedback..." or something. Otherwise, everything LGTM.